web/src/components/atoms/StatusChip/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { ChipColors } from '@/utils/constants/statusChipColor';
+import { Chip, ChipProps } from '@mui/material';
+import { FC } from 'react';
+
+const StatusChip: FC<ChipProps> = ({ label }: ChipProps) => {
+  return (
+    <Chip
+      label={label}
+      sx={{
+        backgroundColor: ChipColors[label as string] ?? '#65707b33',
```

Comment:
Please define every hex value colors, so that it would be easily identified by developers.